spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -763,8 +787,14 @@ function sendPacket(
     // check that the channel must be OPEN to send packets;
     abortTransactionUnless(channel !== null)
     abortTransactionUnless(channel.state === OPEN)
+
     connection = provableStore.get(connectionPath(channel.connectionHops[0]))
     abortTransactionUnless(connection !== null)
+    
+    client = provableStore.get(clientStatePath(connection.clientIdenfier))
```

Comment:
You could do here `clientState = queryClientState(connection.clientIdenfier)` (`queryClientState` is defined in ICS 02)

Reply:
Gotcha, thank you @crodriguezvega! Please, verify if the latest commit [c1637a9](https://github.com/cosmos/ibc/pull/1051/commits/c1637a98afe3220a86a9a443dbd8adba2cfdcac6) address all of your comments.